shopinvader_delivery_carrier/models/delivery_carrier.py

```diff
@@ -2,12 +2,15 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from contextlib import contextmanager
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class DeliveryCarrier(models.Model):
     _inherit = "delivery.carrier"
 
+    description = fields.Text()
```

Comment:
Wouldn't be typical for a simple module delivery_carrier_description ?

Reply:
A module with only a text field ?

Reply:
@rousseldenis ?

Reply:
That exists in multiple repositories. But I can admit it's a little excessive.